release/nightly_tests/dataset/ray_sgd_training.py

```diff
@@ -529,9 +531,13 @@ def train(self):
     preprocessing_end_time = timeit.default_timer()
     print("Preprocessing time (s): ", preprocessing_end_time - e2e_start_time)
 
-    num_columns = len(train_dataset.schema().names)
-    # remove label column and internal Arrow column.
-    num_features = num_columns - 2
+    # filter label column and internal Arrow column.
```

Comment:
Explicitly mention the column name here? `__index_level_0__`

Also in the PR description it says `index_level_0`.